pkgs/by-name/uu/uutils-coreutils/package.nix

```diff
@@ -39,11 +47,20 @@ stdenv.mkDerivation (finalAttrs: {
       "CARGO=${lib.getExe cargo}"
       "PREFIX=${placeholder "out"}"
       "PROFILE=release"
+      "SELINUX_ENABLED=${if selinuxSupport then "1" else "0"}"
       "INSTALLDIR_MAN=${placeholder "out"}/share/man/man1"
     ]
     ++ lib.optionals (prefix != null) [ "PROG_PREFIX=${prefix}" ]
     ++ lib.optionals buildMulticallBinary [ "MULTICALL=y" ];
 
+  env = {
+    SELINUX_INCLUDE_DIR = ''${libselinux.dev}/include'';
+    SELINUX_LIB_DIR = lib.makeLibraryPath [
+      libselinux
+    ];
+    SELINUX_STATIC = "0";
+  };
+
```

Comment:
Don't we want to make this conditional on `seLinuxSupport`?

Reply:
https://github.com/uutils/coreutils/issues/7996

Reply:
Thanks @r-vdp !